File: test_loop/loop.f90
      PROGRAM TEST
      IMPLICIT NONE 
      INTEGER :: I,J,N 
      REAL :: START,FINAL
      DOUBLE PRECISION :: SUM,ADD
      
      CALL CPU_TIME(START)
      
      N=100000
      SUM=0.0
      ADD=1.0
      
      DO I=1,N
         DO J=1,N
           SUM=SUM+ADD
         END DO
      END DO
      
      CALL CPU_TIME(FINAL)
      
      WRITE(*,*) SUM
      WRITE(*,*) "CPU TIME (IN SECONDS):", (FINAL-START)
      WRITE(*,*)
      
      END PROGRAM
      
      

File: test_loop/loop.cpp
#include <iostream>
#include <time.h>
using namespace std;

int main() {
  int i,j,n;
  double sum,add;
  
  clock_t start = clock();
  
  n=100000;
  sum=0;
  add=1;
  
  for (i=1; i<=n; i++){
    for (j=1; j<=n; j++) {  
      sum+=add;    
    }    
  }    
  
  clock_t end = clock();
  
  double elapsed = double(end - start)/CLOCKS_PER_SEC;
  
  cout << sum << endl;
  cout << "CPU TIME (IN SECONDS): " << elapsed << endl;
    
return 0;
}
